llpc/tool/llpcCompilationUtils.h

```diff
@@ -104,11 +104,11 @@ void *VKAPI_CALL allocateBuffer(void *instance, void *userData, size_t size);
 void cleanupCompileInfo(CompileInfo *compileInfo);
 
 // GLSL compiler, compiles GLSL source text file (input) to SPIR-V binary file (output).
-llvm::Expected<std::string> compileGlsl(const std::string &inFilename, ShaderStage *stage,
-                                        const std::string &defaultEntryTarget);
+llvm::Expected<BinaryData> compileGlsl(const std::string &inFilename, ShaderStage *stage,
+                                       const std::string &defaultEntryTarget);
 
 // SPIR-V assembler, converts SPIR-V assembly text file (input) to SPIR-V binary file (output).
```

Comment:
The comments are outdated (it’s not an output *file* anymore)

Reply:
Done